packages/lesswrong/server/resolvers/postResolvers.ts

```diff
@@ -350,9 +367,38 @@ augmentFieldsDict(Posts, {
       },
     },
   },
+
+  glossary: {
+    resolveAs: {
+      type: GraphQLJSON,
+      resolver: async (post: DbPost, args: void, context: ResolverContext) => {
+        // Forum-gating/beta-gating is done here, rather than just client side,
+        // so that users don't have to download the glossary if it isn't going
+        // to be displayed.
+        if (!userCanViewJargonTerms(context.currentUser)) {
+          return;
+        }
+        const jargonTerms = await context.JargonTerms.find({postId: post._id, rejected: false, deleted: false}).fetch();
+
+        return jargonTerms.map((jargonTerm: DbJargonTerm) => ({
+          term: jargonTerm.term,
+          altTerms: jargonTerm.altTerms,
+          html: sanitize(jargonTerm.contents?.html ?? ""),
+        }));
+      }
+    }
+  }
 })
 
 
+declare global {
+  type GlossaryTerm = {
+    term: string
+    altTerms: string[]
+    html: string
+  }
+}
+
```

Comment:
If you do [this](https://github.com/ForumMagnum/ForumMagnum/pull/9771/files#r1800201517), you can delete this